src/QMCTools/convert4qmc.cpp

```diff
@@ -55,10 +55,23 @@ int main(int argc, char **argv)
   QMCGaussianParserBase *parser=0;
   int iargc=0;
   std::string in_file(argv[1]);
+
+  
   std::string punch_file;
   std::string psi_tag("psi0");
   std::string ion_tag("ion0");
-  std::string prefix("sample");
+  std::string prefix(argv[2]);
```

Comment:
This is not safe, based on the help info. The first argument can be omitted and argv[2] is null.

Reply:
The first argument omitted never works. 
Maybe it is time to remove that option and make a clean fail if you don't provide the first argument.

Reply:
I tried it. It works on our test cases.

Reply:
In all cases, I "secured" it for now.